api/python/cellxgene_census/src/cellxgene_census/experimental/_embedding.py

```diff
@@ -136,3 +136,48 @@ def get_embedding(
             np.put(embedding.reshape(-1), indices, emb)
 
     return embedding
+
+
+def get_all_available_embeddings(census_version: str) -> list[dict[str, Any]]:
+    """Return a dictionary of all available embeddings for a given Census version.
+
+    Args:
+        census_version:
+            The Census version tag, e.g., ``"2023-12-15"``.
+
+    Returns:
+        A list of dictionaries, each containing metadata describing an available embedding.
```

Comment:
Options:
1. Return a subset of the metadata that only has relevant information (name, organism, etc). The example listed here is only for reference
2. Return the full metadata.

Reply:
strongly prefer 1) and having a `verbose` argument